### PR TITLE
net/rsyslog: upgrade to 8.17.0

### PR DESCRIPTION
**Pull request by dubek**
Add `libs/libfastjson` which is now a dependency for rsyslog (since 8.17.0).

Upgrade rsyslog to 8.17.0.

Signed-off-by: Dov Murik \<dmurik@us.ibm.com>

### CODE REVIEW

File: net/rsyslog/Makefile [hunk 1]
```diff
@@ -37,6 +37,7 @@ define Package/rsyslog/conffiles
 endef
 
 CONFIGURE_ARGS+= \
+	--enable-omprog \
```
Comment (acinonyx) [inline net/rsyslog/Makefile]: Why do you enable the 'omprog' module in this version?

Comment (dubek) [inline net/rsyslog/Makefile]: It actually should be separated to a another commit and probably be configurable in a submenu. I'll remove for now.

**Comment by dubek** [comment]
Commit 2b59c7c88ec57b93a9524e7e2765324e3dcd594e removes the `enable-omprog` addition. Might be added later separately (as a configurable checkbox).

File: net/rsyslog/Makefile (hunk 1 above)
Comment (acinonyx) [inline net/rsyslog/Makefile]: Ok, thanks @dubek

**Comment by acinonyx** [comment]
You had forgotten to sign-off your commits but now it's too late. I've already merged it without noticing it.

**Comment by dubek** [comment]
I signed-off the PR. I probably misread the instructions. Sorry.

**Pull request merged**